web/src/components/layout/layout.tsx

```diff
@@ -16,7 +16,7 @@ import {
   isStaging,
   replacePathLocale,
 } from '../../utility';
-import { LocaleLink, LocaleNavLink } from '../locale-helpers';
+import { LocaleLink, LocaleNavLink, useLocale } from '../locale-helpers';
```

Comment:
This is the only change in this file, is this still being used?

Reply:
Ah, good catch! Will remove it, thank you.

Reply:
https://github.com/mozilla/voice-web/commit/9df4a5f58693c34da7a123a4b3c92dd1e95db330

(We should make a lint rule for this once the dust settles)